Sources/Sentry/SentryClient.m

```diff
@@ -634,6 +643,29 @@ - (void)recordLost:(BOOL)eventIsNotATransaction reason:(SentryDiscardReason)reas
     }
 }
 
+- (void)attachScreenshots:(SentryScope *)scope
```

Comment:
`l`: Maybe we can put this logic into an extra class. The client is already huge. It would also be easier for testing.

Reply:
No problem. I will do it.